quick_sort/quick.cpp

```diff
@@ -0,0 +1,36 @@
+#include<bits/stdc++.h>
```

Comment:
This is pretty neat!
You should also write some comments!
Check other codes for reference!